backend/src/main/java/me/performancereservation/domain/settlement/SettlementService.java

```diff
@@ -0,0 +1,125 @@
+        LocalDateTime latestSchedule = schedules.stream()
+                .map(PerformanceSchedule::getStartTime)
+                .max(LocalDateTime::compareTo)
+                .orElseThrow(() -> ErrorCode.PERFORMANCE_NOT_FOUND.domainException("공연 스케줄이 존재하지 않습니다."));
+
+        // 정산 신청 가능 날짜 체크 (7일 이내)
+        if (latestSchedule.plusDays(7).isAfter(LocalDateTime.now())) {
+            throw ErrorCode.INVALID_SETTLEMENT_REQUEST.domainException("공연 종료 후 7일이 지나야 정산 신청이 가능합니다.");
+        }
+
+        // 총 정산 금액 계산
+        int totalAmount = calculateTotalAmount(schedules, performance);
+
+        // Settlement 객체 생성
+        // settledAt은 아직 값 설정하지 않고 나중에 confirm 할 때 설정
+        Settlement settlement = Settlement.builder()
+                .performanceId(request.performanceId())
+                .totalAmount(totalAmount)
+                .account(request.account())
+                .bank(request.bank())
+                .status(SettlementStatus.PENDING)
+                .build();
+
+        return settlementRepository.save(settlement).getId();
+    }
+
+    private int calculateTotalAmount(List<PerformanceSchedule> schedules, Performance performance) {
+        int price = performance.getPrice();
+        int totalSeats = performance.getTotalSeats();
+
+        // 스케쥴 리스트로 총 정산금액 누적 계산
+        return schedules.stream()
+                .filter(schedule -> !schedule.isCanceled()) // 취소된 스케쥴은 계산하지 않음
+                .mapToInt(schedule -> price * (totalSeats - schedule.getRemainingSeats()))
+                .sum();
+    }
+
+    @Transactional
+    public SettlementResponse confirmSettlement(Long settlementId) {
+        // 정산 객체 조회
+        Settlement settlement = settlementRepository.findById(settlementId)
+                .orElseThrow(() -> ErrorCode.SETTLEMENT_NOT_FOUND.domainException("존재하지 않는 정산입니다."));
+
+        // 공연 정보 조회
+        Performance performance = performanceRepository.findById(settlement.getPerformanceId())
+                .orElseThrow(() -> ErrorCode.PERFORMANCE_NOT_FOUND.domainException("존재하지 않는 공연입니다."));
+
+        // 정산 상태 변경 및 완료 시간 설정
+        settlement.confirm();
+
+        // SettlementResponse 생성 및 반환
+        return SettlementResponse.fromEntity(settlement, performance.getTitle());
+    }
+
+    @Transactional(readOnly = true)
+    public Page<SettlementResponse> findAllSettlementsWithUserId(Long userId, Pageable pageable) {
+        return settlementRepository.findAllSettlementsWithUserId(userId, pageable);
+    }
+
+    @Transactional(readOnly = true)
+    public Page<SettlementResponse> findAllSettlements(Pageable pageable) {
+        return settlementRepository.findAllSettlements(pageable);
+    }
+
+    @Transactional(readOnly = true)
+    public Page<SettlementResponse> findAllSettlementsByStatus(String status, Pageable pageable) {
+
+        SettlementStatus settlementStatus = getSettlementStatus(status);
+        return settlementRepository.findAllSettlementsByStatus(settlementStatus, pageable);
+    }
```

Comment:
컨트롤러 분기 처리를 여기서 수행하게 하면 어떤가 였습니다!
```suggestion
    @Transactional(readOnly = true)
    public Page<SettlementResponse> findAllSettlements(Pageable pageable, status) {
        if(status == null) {
            return settlementRepository.findAllSettlements(pageable);
        }
        
        SettlementStatus settlementStatus = getSettlementStatus(status);
        return settlementRepository.findAllSettlementsByStatus(settlementStatus, pageable);
    }
```